### PR TITLE
refactor(components): progress ring size API

### PR DESCRIPTION
**Pull request by yinonov**

### CODE REVIEW

**Review by YonatanKra — DISMISSED** [review]
LGTM but maybe @rinaok or @rachelbt should take another look?

File: libs/components/src/lib/progress-ring/README.md [hunk 1]
```diff
@@ -88,3 +105,6 @@ Remove `value` or set it to be a non-number value (e.g. `indeterminate`) to show
         <vwc-progress-ring min="0" max="50" value="indeterminate"></vwc-progress-ring>
     </vwc-layout>
 ```
+## CSS Variables
+
+### TODO sizes variable
```
Comment (yinonov) [inline libs/components/src/lib/progress-ring/README.md:110]: Should document the size css variable

File: libs/components/src/lib/progress-ring/progress-ring.scss [hunk 2]
```diff
@@ -6,22 +8,31 @@
...
+	height: var(#{variables.$vvd-progress-ring-size}, map.get($sizes, medium));
 	align-items: center;
 	outline: none;
 }
-.base.base-large {
-	width: 48px;
-	height: 48px;
+
+.base.size-small {
+  #{variables.$vvd-progress-ring-size}: 24px;
```
Comment (yinonov) [inline libs/components/src/lib/progress-ring/progress-ring.scss:27]: On 2nd thought, this is not our common pattern. We usually override if css variable is defined

File: libs/components/src/lib/progress-ring/README.md [hunk 3]
```diff
@@ -88,3 +88,56 @@ Remove `value` or set it to be a non-number value (e.g. `indeterminate`) to show
         <vwc-progress-ring min="0" max="50" value="indeterminate"></vwc-progress-ring>
     </vwc-layout>
 ```
+## CSS Custom Properties
```
Comment (YonatanKra) [inline libs/components/src/lib/progress-ring/README.md:91]: Why have we reverted to css variable?

File: libs/components/src/lib/progress-ring/progress-ring.scss [hunk 4]
```diff
@@ -6,23 +8,16 @@
 );
 @use '../../../../shared/src/lib/sass/mixins/connotation' as connotation;
 
+
 .base {
 	@include connotation.connotation;
+  $size: calc(clamp(4, var(#{variables.$vvd-progress-ring-density}, 9), 12) * 4px);
```
Comment (YonatanKra) [inline libs/components/src/lib/progress-ring/progress-ring.scss:14]: Nice one. Didn't know about clamp

File: libs/components/src/lib/progress-ring/README.md (hunk 3 above)
Comment (yinonov) [reply libs/components/src/lib/progress-ring/README.md:91]: It can be a property really. Will refactor

**Review by YonatanKra — APPROVED** [review]

**Pull request merged**